include/shark/Models/Trees/CARTClassifier.h

```diff
@@ -100,7 +100,7 @@ class CARTClassifier : public AbstractModel<RealVector,LabelType>
 	/// Constructor taking the splitMatrix as argument
 	CARTClassifier(SplitMatrixType const& splitMatrix)
 	{
-		setSplitMatrix(splitMatrix);
+		m_splitMatrix=splitMatrix;
```

Comment:
Is this change here on purpose avoiding the call to optimizeSplitMatrix by setSplitMatrix? What would be the rationale behind that?